README-etn.md

```diff
@@ -15,6 +15,8 @@ There are some other files that have also been modified to support our process.
 * images.CI\linux-and-win\azure-pipelines\image-generation.yml -> image-generation-etn.yml
 * images.CI\linux-and-win\build-image.ps1 -> build-image-etn.ps1
 * Note: images/win/scripts/Tests/WindowsFeatures.Tests.ps1: On line 21 keep $minimumFreeSpaceMB = 5 * 1024 to have $availableSpaceMB greater than $minimumFreeSpaceMB
+* Note: in the windows2019-etn.json (provisioners section), the line "{{ template_dir }}/scripts/Installers/Install-AndroidSDK.ps1" needs to be removed, as we are not installing Android SDK to Get some (~12 GB space) on the VM to have multiple versions of zephyr SDK
```

Comment:
can we mention about removal of `MysqlCli` and `PostgreSQL` as well here ?

Reply:
Added Notes